public/app/core/utils/timezone.ts

```diff
@@ -0,0 +1,24 @@
+import { dateTime } from '@grafana/data';
+
+// This function will try to return the proper full name of the local timezone
+// Chrome does not handle the timezone offset (but phantomjs does)
+export function getLocalTimeZone(): string {
```

Comment:
Feels like this function should be placed in https://github.com/grafana/grafana/blob/bb0a438705688f26f294b21de25dc3770dfc9cdc/packages/grafana-data/src/datetime/timezones.ts